mod_joomcat.xml

```diff
@@ -2,12 +2,12 @@
 <extension type="module" version="3.0" client="site" method="upgrade">
   <name>mod_joomcat</name>
   <author>JoomGallery::Project Team</author>
-  <creationDate>2019/04/03</creationDate>
+  <creationDate>2019/07/09</creationDate>
```

Comment:
Should be 2020/...? 
Furthermore:
I have tested this item ✅ successfully. Thank you very much.

Reply:
Yep, I'll correct that in the release commit. Thank you!